File: C#/TripTalk/TripTalk/Clases/ViajeInteres.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripTalk.Clases
{

        public class ViajeInteres
        {
            int idUsuario;
            int idPublicacion;

            public ViajeInteres()
            {
            }

            public ViajeInteres(int idUsuario, int idPublicacion)
            {
                this.idUsuario = idUsuario;
                this.idPublicacion = idPublicacion;
            }

            [JsonProperty("idUsuario")]
            public int IdUsuario
            {
                get => idUsuario;
                set => idUsuario = value;
            }

            [JsonProperty("idPublicacion")]
            public int IdPublicacion
            {
                get => idPublicacion;
                set => idPublicacion = value;
            }

            public override string ToString()
            {
                return "ViajeInteres{" + "idUsuario=" + idUsuario + ", idPublicacion=" + idPublicacion + '}';
            }
        }
    }

File: Java/TripTalk/src/main/java/com/mycompany/triptalk/clases/ViajeInteres.java

package com.mycompany.triptalk.clases;

public class ViajeInteres {
    private int idUsuario;
    private int idPublicacion;

    public ViajeInteres(int idUsuario, int idPublicacion) {
        this.idUsuario = idUsuario;
        this.idPublicacion = idPublicacion;
    }

    public int getIdPublicacion() {
        return idPublicacion;
    }

    public void setIdPublicacion(int idPublicacion) {
        this.idPublicacion = idPublicacion;
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }
    
    
}
